docs/reference/search/rank-eval.asciidoc

```diff
@@ -3,39 +3,90 @@
-This can only be done if the search result quality is evaluated constantly across a representative test suite of typical user queries, so that improvements in the rankings for one particular query doesn't negatively effect the ranking for other types of queries.
+In order to get started with search quality evaluation, three basic things are 
+needed:
 
-In order to get started with search quality evaluation, three basic things are needed:
+. a collection of documents you want to evaluate your query performance against, 
+  usually one or more indices.
+. a collection of typical search requests that users enter into your system.
+. a set of document ratings that judge the documents relevance with respect to a 
+  search request.
+  
+It is important to note that one set of document ratings is needed per test 
+query, and that the relevance judgements are based on the information need of 
+the user that entered the query.
 
-. a collection of documents you want to evaluate your query performance against, usually one or more indices
-. a collection of typical search requests that users enter into your system
-. a set of document ratings that judge the documents relevance with respect to a search request+
-  It is important to note that one set of document ratings is needed per test query, and that
-  the relevance judgements are based on the information need of the user that entered the query.
+The ranking evaluation API provides a convenient way to use this information in 
+a ranking evaluation request to calculate different search evaluation metrics. 
+This gives a first estimation of your overall search quality and give you a 
+measurement to optimize against when fine-tuning various aspect of the query 
+generation in your application.
 
-The ranking evaluation API provides a convenient way to use this information in a ranking evaluation request to calculate different search evaluation metrics. This gives a first estimation of your overall search quality and give you a measurement to optimize against when fine-tuning various aspect of the query generation in your application. 
 
-[float]
-==== Ranking evaluation request structure
+[[search-rank-eval-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
+
+
+[[search-rank-eval-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
++
+--
+Defaults to `open`.
+--
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
+
+
+[[search-rank-eval-api-example]]
+==== {api-examples-title}
 
 In its most basic form, a request to the `_rank_eval` endpoint has two sections:
 
-[source,js]
+[source,console]
```

Comment:
I don't believe you can use `[source,console]` with `// NOTCONSOLE`. This may be causing the CI to fail.
```suggestion
[source,js]
```

Reply:
Yeah, it sounds logical. I hoped that `NOTCONSOLE` will overwrite `console` at the beginning.